plugins/catalog/src/components/CatalogTable/CatalogTable.tsx

```diff
@@ -62,6 +64,38 @@ const YellowStar = withStyles({
   },
 })(Star);
 
+const stringifySortingEntityRef = (ref: Entity): string => {
+  const kind = ref.kind;
+  const namespace = ref.metadata.namespace ?? DEFAULT_NAMESPACE;
+  const name = ref.metadata.title || ref.metadata.name;
```

Comment:
Hm. Does it really make sense to keep kind and namespace in the sorting, when a title is present? When the title is present, it's _the only_ thing that's rendered in the column, right? So I'd expect to just return that if it exists, and otherwise return the stringifyEntityRef(ref)

Reply:
I finally tried with `humanizeEntityRef,` the same way it's implemented in the custom sorting